client/components/editor/Sidebar/index.vue

```diff
@@ -64,9 +67,14 @@ export default {
     selectedActivity: { type: Object, required: true },
     selectedElement: { type: Object, default: null }
   },
-  data: () => ({ selectedTab: 'browser', commentCount: 0 }),
+  data: () => ({ selectedTab: 'browser', commentCount: 0, unseenCount: 0 }),
```

Comment:
You can get rid of comment count since it is no longer used. Since this is sidebar root rename unseenCount to unseenComments (store items and get length for count).